### ISSUE TITLE
Not All Relations are Equal: Mining Informative Labels for Scene Graph Generation

### ISSUE THREAD

**Issue by SerialLain3170** [SerialLain3170/nagato#87]
## Overview
- Authors propose a method to deal with sparse predicates by dividing labels into implicit relations and explicit ones
  - As for implicit labels, authors train the model with cross entropy
  - When it comes to explicit labels, authors create target labels by using the model trained with explicit labels. After that, they train the model with Kullback-Leibler divergence.
  - To avoid overfitting to imputed labels, authors use Manifold Mixup

<img width="520" alt="Screen Shot 2021-12-02 at 19 02 55" src="https://user-images.githubusercontent.com/32360147/144400459-dc7da7d6-9cfa-4b84-a1c2-299e9aa39134.png">

**Comment by SerialLain3170** [comment]
## Evaluation
- Previous methods trained with the method are superior to ones without the method

<img width="921" alt="Screen Shot 2021-12-02 at 19 11 31" src="https://user-images.githubusercontent.com/32360147/144401790-b4023dc7-ad05-4b76-9770-e2e7491df046.png">

**Comment by SerialLain3170** [comment]
## Supplemental Information

[Paper Link](https://arxiv.org/pdf/2111.13517.pdf)

### Authors
Arushi Goel, Basura Fernando, Frank Keller, Hakan Bilen

### Organization
School of Informatics, University of Edinburgh, UK
Institute of High Performance Computing, A*STAR, Singapore

### Conference
arXiv preprint 2021
